Apps/WebClient/src/ClientApp/src/store/modules/user/actions.ts

```diff
@@ -241,4 +250,27 @@ export const actions: ActionTree<UserState, RootState> = {
                 });
         });
     },
+    getPatientData(context, params: { hdid: string }): Promise<void> {
+        return new Promise((resolve, reject) => {
```

Comment:
if retrieved, no need to call again. Lets add a check to make sure the store is not loaded before requesting again

Reply:
Added the check for loading Patient Data only once in case we call it multi time.